rskj-core/src/main/java/co/rsk/peg/bitcoin/BitcoinUtils.java

```diff
@@ -1,14 +1,20 @@
 package co.rsk.peg.bitcoin;
 
 import co.rsk.bitcoinj.core.BtcTransaction;
+import co.rsk.bitcoinj.core.ScriptException;
 import co.rsk.bitcoinj.core.Sha256Hash;
 import co.rsk.bitcoinj.core.TransactionInput;
 import co.rsk.bitcoinj.script.Script;
-import co.rsk.peg.BridgeUtils;
+import co.rsk.bitcoinj.script.ScriptChunk;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
+import java.util.List;
 import java.util.Optional;
 
 public class BitcoinUtils {
+
+    private static final Logger logger = LoggerFactory.getLogger("BitcoinUtils");
```

Comment:
```suggestion
    private static final Logger logger = LoggerFactory.getLogger(BitcoinUtils.class);
```
Avoids possible typos, like it happened in PegUtilsLegacy

Reply:
Updated.